docs/source/algorithms/RemoveInstrumentGeometry-v1.rst

```diff
@@ -0,0 +1,47 @@
+
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Remove Instrument Geometry from workspace.
+
+
+Usage
+-----
+
+**Example - RemoveInstrumentGeometry**
+
+.. testcode:: RemoveInstrumentGeometryExample
+
+    # create some workspace with an instrument
+    ws = CreateSampleWorkspace()
+    print("Instrument Geometry exists:  {}".format(!ws.getInstrument().isEmptyInstrument()))
+
+    # delete instrument geometry
+    RemoveInstrumentGeometry(ws)
+    print("Instrument Geometry exists (should be false):  {}".format(!ws.getInstrument().isEmptyInstrument()))
+
+.. testcleanup:: RemoveInstrumentGeometry
```

Comment:
RemoveInstrumentGeometry should be RemoveInstrumentGeometryExample, to match the other labels